docs/Components.md

```diff
@@ -150,3 +150,17 @@ This is an enhanced version of the built-in React Native Text component. It adds
 ### TextField
 
 ### Wallpaper
+
+This component renders an image background to a component or a screen. 
+
+```tsx
+ <Wallpaper
+ style={{ 
+    marginHorizontal: 10,
+    paddingVertical: 10,
+  }} 
+ backgroundImage={$newImage}
```

Comment:
This shouldn't have a $ on it, as that's reserved for styles.

Indentation isn't right on this, too.